mariadb/stack.yml

```diff
@@ -1,16 +1,15 @@
 # Use root/example as user/password credentials
-version: '3.1'
 
 services:
 
   db:
-    image: mariadb
+    image: mariadb:11.7
     restart: always
     environment:
       MARIADB_ROOT_PASSWORD: example
 
   adminer:
-    image: adminer
+    image: adminer:5.0.1
```

Comment:
while I do generally like the idea of pinning versions, on a UI is it important? With this versioned pinned the updates to this are going to be quite frequent - current adminer versions: https://github.com/docker-library/official-images/blob/master/library/adminer